docs/source/partial-transformers/cats-integration.rst

```diff
@@ -0,0 +1,106 @@
+.. _partial-cats-integration:
+
+Cats integration
+================
+
+Chimney provides `Cats <https://typelevel.org/cats>`_ library integration module
+for partial transformers.
+To include it to your SBT project, add the following line to your ``build.sbt``:
```

Comment:
Formally sbt is capitalized as "sbt" (all lower case), so we might want to use that in docs.